ui/src/components/shared/AutocompleteSearchBar.vue

```diff
@@ -177,10 +177,20 @@ function select(event: KeyboardEvent) {
 async function onEnter(event: any) {
   if (resultsOP.value) resultsOP.value.show(event);
   if (searchText.value) {
-    results.value = await search(searchText.value, undefined, { pageNumber: 1, pageSize: 10 }, props.osQuery, props.imQuery);
+    results.value = await search();
   }
 }
 
+async function search() {
+  searchLoading.value = true;
+  const imQueryCopy = cloneDeep(props.imQuery) ?? { query: {} };
```

Comment:
does cloneDeep handle undefined inputs as expected?
Turnary might be better
props.imQuery ? cloneDeep(props.imQuery) : {query:{}}